source/blog/2016-08-30-new-centos-atomic-host-with-package-layering-support.html.md

```diff
@@ -0,0 +1,60 @@
+---
+title: New CentOS Atomic Host with Package Layering Support
+author: jbrooks
+date: 2016-08-30 18:38:04 UTC
+tags: centos, rpm-ostree
+published: false
+comments: true
+---
+
+Last week, the CentOS Atomic SIG [released](https://seven.centos.org/2016/08/announcing-a-new-release-of-centos-atomic-host-2/) an updated version of CentOS Atomic Host (tree version 7.20160818), featuring support for rpm-ostree package layering. 
+
+CentOS Atomic Host is available as a VirtualBox or libvirt-formatted Vagrant box, or as an installable ISO, qcow2 or Amazon Machine image. Check out the [CentOS wiki](https://wiki.centos.org/SpecialInterestGroup/Atomic/Download) for download links and installation instructions, or read on to learn more about what’s new in this release.
+
+CentOS Atomic Host includes these core component versions:
+
+* docker-1.10.3-46.el7.centos.10.x86_64
+* kubernetes-1.2.0-0.13.gitec7364b.el7.x86_64
+* kernel-3.10.0-327.28.2.el7.x86_64
+* atomic-1.10.5-7.el7.x86_64
+* flannel-0.5.3-9.el7.x86_64
+* ostree-2016.7-2.atomic.el7.x86_64
+* etcd-2.3.7-2.el7.x86_64
+* cloud-init-0.7.5-10.el7.centos.1.x86_64
+
+## Package Layering
+
+Using the command rpm-ostree pkg-add, it’s now possible to layer new packages into an installed image that persist across reboots and upgrades, a topic that [Jonathan Lebon](https://github.com/jlebon) [covered in some detail](http://www.projectatomic.io/blog/2016/07/hacking-and-extending-atomic-host/) in a post last month.
+
+For instance, if I wanted to install ansible on an atomic host:
+
+```
+# rpm-ostree pkg-add epel-release
+# reboot
+# rpm-ostree pkg-add ansible
+# reboot
+# ansible --version
+ansible 2.1.1.0
+  config file = /etc/ansible/ansible.cfg
+  configured module search path = Default w/o overrides
+```
+
+I first installed the `epel-release` package because ansible lives in EPEL. The intermediate reboot was required to boot into the new EPEL-i-fied tree. I could have instead added the repo file for EPEL in my `/etc/yum.repos.d/` directory, and skipped the extra install and reboot operations.
```

Comment:
There is also work upstream on https://bugzilla.gnome.org/show_bug.cgi?id=767977